packages/telemetry/src/slog-to-otel.js

```diff
@@ -128,16 +189,7 @@ export const makeSlogToOtelKit = (tracer, overrideAttrs = {}) => {
       }
       case 'send': {
         name = 'message';
-        // TODO: The arguments to a method call can be pretty big.
-        delete attrs.msg;
-        const { methargs, method = extractMethod(methargs) } = message.msg;
-        const slots =
-          message.msg.methargs?.slots ?? message.msg.args.slots ?? [];
-        attrs.details = `E(${message.target}).${method}`;
-        attrs['msg.target'] = message.target;
-        attrs['msg.method'] = method;
-        attrs['msg.slots'] = slots.join(',');
-        attrs['msg.result'] = message.msg.result;
+        assign(attrs, formatMsg(message.target, parseMsg(message.msg)));
```

Comment:
this change from `{'msg.target': ...}` to `{msg: {target: ...}}` is intended, right?

Reply:
There is actually no change, the serializer this goes through results in the same output